packages/desktop/views/dashboard/collectibles/components/CollectionsGallery.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import CollectionsGalleryItem from './CollectionsGalleryItem.svelte'
+    import { Collection, Collections } from '@core/nfts'
+    import VirtualList from '@sveltejs/svelte-virtual-list'
+    import { breakpoint } from '@core/app/stores'
+
+    export let collections: Collections
+
+    $: collectionsArray = Object.values(collections)
```

Comment:
Look at my comment on `CollectionGalleryView`

```suggestion
    export let collections: Collection[]
```